src/probes.ts

```diff
@@ -46,6 +46,7 @@ const counters = {
 
 export function incomingMessage (topic: string, message: Message) {
     log.trace({ message }, 'incoming message');
+    log.info({ message }, 'incoming message');
```

Comment:
I can't figure out where to see the `trace` log messages but this would be a helpful one to see in the logs too so adding an `info` log message.

Reply:
'trace' is a logging level.  It's set in deployment/remediations-consumer-clowdapp.yaml but can be overridden in the app-interface config (it's currently set to 'info').  There's also a config parameter that controls kafaka logging, which appears to be disabled so we might want to turn that on in stage to see what we get.  🙂